Sources/main.cpp

```diff
@@ -310,6 +310,10 @@ int main(int argc, char **argv) {
   opts.addOption(ds2::OptParse::boolOption, "setsid", 'S',
                  "make ds2 run in its own session");
 
+  // gdb-server compatibility options
+  opts.addOption(ds2::OptParse::boolOption, "once", 'O',
+                 "exit after one execution of inferior", true);
```

Comment:
Nit: you should add ` (no-op)` or even ` (default)` at the end of the description string.